src/mpyl/stages/discovery.py

```diff
@@ -283,19 +282,21 @@ def create_run_plan(
             logger.debug(f"Run plan: {existing_run_plan}")
         return existing_run_plan
 
-    run_plan = _discover_run_plan(
-        logger=logger,
-        repository=repository,
-        all_projects=all_projects,
-        all_stages=all_stages,
-        selected_projects=selected_projects,
-        selected_stage=selected_stage,
-        tag=tag,
-        changed_files_path=changed_files_path,
-    )
+    if changed_files_path:
```

Comment:
This seems weird considering the pr title 😬 why is this needed? You can make the field required in the schema so the config won't even load if it's not set